packages/owl-bot/src/copy-code.ts

```diff
@@ -118,20 +118,22 @@ ${err}`,
     logger.error(`Created issue ${issue.data.html_url}`);
     return; // Success because we don't want to retry.
   }
-  sourceRepoCommitHash = await copyCode(
+  const {sourceCommitHash, commitMsgPath} = await copyCode(
     sourceRepo,
     sourceRepoCommitHash,
     destDir,
     workDir,
     yaml,
     logger
   );
+  cmd('git add -A', {cwd: destDir});
+  cmd(`git commit -F "${commitMsgPath}" --allow-empty`, {cwd: destDir});
```

Comment:
I'm not following how this does what the title of this PR says (not committing changes).

Reply:
These git commands were moved from the `copyCode()` function into the `copyCodeAndCreatePullRequest()` function.

The [copy-code command](https://github.com/googleapis/repo-automation-bots/blob/master/packages/owl-bot/src/bin/commands/copy-code.ts) invokes `copyCode()` directly, so it will no longer execute the git commands.  The hourly cron job invokes `copyCodeAndCreatePullRequest()` so it will continue committing changes and creating pull requests.